test/method/testcda.py

```diff
@@ -11,21 +11,24 @@
 """Test the CDA method in cclib"""
 
 from __future__ import print_function
+
+import sys
 import os
 import logging
 import unittest
 
 import numpy
 
-from testall import getfile
+sys.path.append("..")
+from test_data import getdatafile

```

Comment:
Move getfile and getdatafile to the parent dir?

On Mon, Jun 13, 2016 at 2:43 AM, Eric Berquist <notifications@github.com>
wrote:

> In test/method/testcda.py
> <https://github.com/cclib/cclib/pull/280#discussion_r66731016>:
>
> >  import os
> >  import logging
> >  import unittest
> >
> >  import numpy
> >
> > -from testall import getfile
> > +sys.path.append("..")
> > +from test_data import getdatafile
>
> I would welcome a better way of doing this.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cclib/cclib/pull/280/files/6698d2ea901e60bed409a6d3290a0a1ec01be845#r66731016>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAtZErVK8-myaS3-TaW0qj3WEPo6Kcjwks5qLKeggaJpZM4Iz7Nx>
> .
>